### ISSUE TITLE
Notification On-click Event

### ISSUE THREAD

**Issue by Krystille** [sbondini/Xamarin.LocalNotifications#12]
How to add an on-click event for notifications? Or is there a way to handle this? I am using xamarin forms.

**Comment by SrCristian** [comment]
I have the same question. Did you resolve it?

**Comment by Krystille** [comment]
Nope. I haven't implemented it. I think you need to use pending intent for this.
